domains/istiyak.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Istiyak personal developer website",
+    "repo": "https://github.com/istiyakaminsnato",
+    "owner": {
+        "username": "istiyakaminsanto",
+        "email": "istiyakaminsanto@gmail.com"
+    },
+    "record": {
+        "CNAME": "pafg.com.bd",
```

Comment:
This doesn't seem to be a legitimate use of this service. This project is for personal websites for developers